packages/core-flows/src/product/workflows/delete-product-variants.ts

```diff
@@ -1,12 +1,17 @@
 import { WorkflowData, createWorkflow } from "@medusajs/workflows-sdk"
-import { deleteProductVariantsStep } from "../steps"
+import {
+  deleteProductVariantsStep,
+  removeVariantPricingLinkStep,
+} from "../steps"
 
 type WorkflowInput = { ids: string[] }
 
 export const deleteProductVariantsWorkflowId = "delete-product-variants"
 export const deleteProductVariantsWorkflow = createWorkflow(
   deleteProductVariantsWorkflowId,
   (input: WorkflowData<WorkflowInput>): WorkflowData<void> => {
+    // Question: Should we also remove the price set manually, or would that be cascaded?
```

Comment:
I think when using `delete` through the remoteLink, it soft deletes and cascades.